docker/release/Dockerfile

```diff
@@ -1,15 +1,22 @@
 FROM ubuntu:16.04
 
 RUN apt-get update; \
-    apt-get install -y libc-ares-dev
+    apt-get install -y libc-ares-dev jq
+
+COPY wait-for-it.sh run-iroha.sh go-grpc-health /
 
 #Install iroha
 COPY iroha.deb /tmp/iroha.deb
 RUN apt-get install -y /tmp/iroha.deb; \
-    rm -f /tmp/iroha.deb
+    rm -f /tmp/iroha.deb; \
+    chmod +x /run-iroha.sh; \
+    chmod +x /wait-for-it.sh; \
+    chmod +x /go-grpc-health
 
 WORKDIR /opt/iroha_data
 
-COPY entrypoint.sh /entrypoint.sh
-ENTRYPOINT ["/entrypoint.sh"]
-CMD ["/sbin/init"]
+HEALTHCHECK \
+  --interval=10s --timeout=3s --retries=6 \
```

Comment:
Retries num is quite high imo, 3 is enough
And since it's a local container timeout also can be decreased

Reply:
You're wrong. Iroha starts 3 sec on my Mac, but in Travis ci it starts 40 sec or so